src/components/auth/login.js

```diff
@@ -58,9 +59,13 @@ class LoginPage extends React.Component {
         localStorage.setItem("token", res.data.token);
         // updates token to refresh navbar
         this.props.updateToken(localStorage.getItem("token"));
+        // sets up userid in app
+        this.props.updateUserid(res.data.id);
         //redirect to Groas dashboard page
         if (this.state.user.user_name && this.state.user.password) {
-          this.props.history.push("/dashboard", { userid: res.data.id });
+          this.props.history.push(`/${res.data.id}/recommended`, {
+            userid: res.data.id
```

Comment:
Removed passing history object since we've moved to match object.

Reply:
This is good, don't know if it was related but I was getting undefined in the previous iteration path name, which is not the case on this deployment